scss/index.scss

```diff
@@ -1,20 +1,30 @@
 // Index Page Colors
-$about-background: #f4ecf2 !default;
-$about-title: #8a3c7a !default;
+$about-title: darken(#9c2fb5,5%) !default;
+$about-highlight: lighten($about-title,48%) !default;
 
-$faq-background: #90eee3 !default;
-// dark green for FAQ title
-$faq-title: #1c855f !default;
 
-// gray-900 is not defined here (but in custom.css) - writing manually here
-$faq-content: #212529 !default;
+$industries-title: darken(#4744ac,15%) !default;
+$industries-background: darken(#93d6ee,10%) !default;
+
+$sponsors-title: darken(#199280,2%) !default;
+$sponsors-highlight: lighten($sponsors-title,50%) !default;
+
+$faq-background: #90eee3 !default;
+$faq-title: darken(#171455,0%) !default;
```

Comment:
I think a green FAQ title works better, the industries section has a dark blue title so both need to have that same sort of relationship I guess.

![image](https://user-images.githubusercontent.com/7832610/80066535-bc5e9c80-8501-11ea-9091-8679ef078358.png)

```suggestion
$faq-title: darken(#245a2a,0%) !default;
```